--- a/sound/soc/codecs/Kconfig	2024-04-11 10:10:19.544748160 +0800
+++ b/sound/soc/codecs/Kconfig	2024-04-11 10:10:07.872537410 +0800
@@ -2162,4 +2162,9 @@ config SND_SOC_LPASS_TX_MACRO
 	select SND_SOC_LPASS_MACRO_COMMON
 	tristate "Qualcomm TX Macro in LPASS(Low Power Audio SubSystem)"
 
+config OUT_TO_PINS_CODEC
+	tristate "out_to_pins Speaker Amplifier"
+	default m
+	help
+	  out_to_pins Speaker Amplifier.
 endmenu
